packages/esm-patient-chart-app/src/encounter-list/hooks/usePatientDeathStatus.ts

```diff
@@ -0,0 +1,14 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWRImmutable from 'swr';
+
+export function usePatientDeathStatus(patientUuid: string) {
```

Comment:
Why do we need a hook just to load the patient's death status? This is overkill. The extension will already have the patient fed into it as a prop; use that.